docs/content/ingestion/convert.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: doc_page
+---
+
+The convert task suite takes active segments and will recompress them using a new IndexSpec. This is handy when doing activities like migrating from Concise to Roaring, or adding dimension compression to old segments.
```

Comment:
New users don't have to convert segments and I don't think it makes sense to have its own TOC entry